app/src/main/java/com/codesoom/application/ProductService.java

```diff
@@ -0,0 +1,80 @@
+package com.codesoom.application;
+
+import com.codesoom.domain.Product;
+import com.codesoom.domain.ProductRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * 상품을 관리하기 위한 클래스
+ */
```

Comment:
단순하고 확실하게 적어줍시다. 클래스를 설명하고 있는데 얘는 클래스에요 라고 하면 군더더기라 할 수 있겠죠.
```suggestion
/**
 * 상품을 관리합니다.
 */
```